buildSrc/src/main/groovy/GradlePlugins.groovy

```diff
@@ -0,0 +1,5 @@
+class GradlePlugins {
```

Comment:
The constants should be accessible as `GradlePlugins.bintrayRelease` etc in all the build scripts. I suppose the IDE wasn't auto-completing on those?